spec/providers/firestore.spec.ts

```diff
@@ -29,4 +30,47 @@ describe('providers/firestore', () => {
     expect(snapshot.data()).to.deep.equal(undefined);
     expect(snapshot.id).to.equal('doc-id');
   });
+
+  it('should allow geopoints with makeDocumentSnapshot', () => {
+    const test = fft();
```

Comment:
Optional - could get rid of this line thats repeated in every test cases and use a before/beforeEach instead